### PR TITLE
LWRP Unlit (without graphicTest)

### PR DESCRIPTION
**Pull request by PaulDemeulenaere**
### Purpose of this PR
Public merge request of https://github.cds.internal.unity3d.com/unity/vfx-graph/pull/75/files

---
### Release Notes
We have another MR in progress which is adding dedicated test on katana but locked with yamato configuration : 
https://github.cds.internal.unity3d.com/unity/vfx-graph/pull/102

---
### Testing status
There is a temporary project (the 7z stored with TestProject, will be removed soon) which can help to validate LWRP integration, here result of these test run locally :
image
 
---
### Overall Product Risks
**Technical Risk**: Low
**Halo Effect**: Low

---
### Comments to reviewers
@phi-lira There is a minor change in `com.unity.render-pipelines.lightweight/Runtime/LightweightRenderPipeline.cs` which requires your validation

### CODE REVIEW

File: com.unity.visualeffectgraph/Shaders/RenderPipeline/LWRP/VFXCommon.cginc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    float3 posWS = TransformObjectToWorld(posOS);
+    return VFXTransformPositionWorldToClip(posWS);
+}
+
+float3 VFXTransformPositionWorldToView(float3 posWS)
+{
+    return TransformWorldToView(posWS);
+}
+
+float4x4 VFXGetObjectToWorldMatrix()
```
Comment (phi-lira) [inline com.unity.visualeffectgraph/Shaders/RenderPipeline/LWRP/VFXCommon.cginc]: Why do you need wrapper transform functions?

**Comment by PaulDemeulenaere** [comment]
> why do you mix .cginc with .hlsl?

For consistency with other templates, we have these .cginc since the start of development, we have a task to rename these using only ".hlsl"

File: com.unity.visualeffectgraph/Shaders/RenderPipeline/LWRP/VFXCommon.cginc (hunk 1 above)
Comment (PaulDemeulenaere) [reply com.unity.visualeffectgraph/Shaders/RenderPipeline/LWRP/VFXCommon.cginc]: We handle automatically space conversation in template code, HDRP adds camera relative behavior which requires some special variant, see https://github.com/Unity-Technologies/ScriptableRenderPipeline/blob/master/com.unity.visualeffectgraph/Shaders/RenderPipeline/HDRP/VFXCommon.cginc

Comment (PaulDemeulenaere) [reply com.unity.visualeffectgraph/Shaders/RenderPipeline/LWRP/VFXCommon.cginc]: And we have still an unofficial support of legacy render pipeline

**Comment by PaulDemeulenaere** [comment]
I test manually 08_Shadow with LWRP updating material (screenshot above is wrong)
image

**Pull request merged**